airbyte-integrations/bases/base-typing-deduping-test/src/main/java/io/airbyte/integrations/base/destination/typing_deduping/BaseTypingDedupingTest.java

```diff
@@ -178,6 +193,16 @@ protected JsonNode getConfig() {
     return config;
   }
 
+  /**
+   * Override this method only when skipping T&D and only compare raw tables and skip final table
+   * comparison. For every other case it should always return false.
+   *
+   * @return
+   */
+  protected boolean disableFinalTableComparison() {
```

Comment:
readability nit: prefer positive method name, e.g. `enableFinalTableComparision() { return true; }`

Reply:
I think i went with same naming as actual flag of `disable_type_dedupe`. if strong pref i can switch to +ve naming

Reply:
fair! not a strong preference